pandas/tests/reshape/test_pivot.py

```diff
@@ -1109,6 +1109,20 @@ def test_pivot_margins_name_unicode(self):
         expected = pd.DataFrame(index=index)
         tm.assert_frame_equal(table, expected)
 
+    def test_pivot_func_strings(self):
+        # GH #18713
+        f = lambda func: pivot_table(self.data, values=['D', 'E'],
+                                     index=['A', 'B'], columns='C',
+                                     aggfunc=func)
+        result = f('sum')
+        expected = f(np.sum)
+        tm.assert_frame_equal(result, expected)
+        result = f(['mean', 'std'])
+        means = f(np.mean)
+        stds = f(np.std)
+        expected = concat([means, stds], keys=['mean', 'std'], axis=1)
+        tm.assert_frame_equal(result, expected)
+
```

Comment:
I like what you're testing here, but I would modify a couple of things:

1) Add newlines after each `tm.assert_frame_equal`.  It makes it easier to read.
2) Construct the output explicitly using the `DataFrame` constructor (and check against that too).  At this point, I know for example that `f('sum')` and `f(np.sum)` should be the same, but I don't know what the actual result is.  e.g.

~~~python
result = f('sum')
func_result = f(np.sum)
expected = DataFrame(...)
tm.assert_frame_equal(result, func_result)  # consistency
tm.assert_frame_equal(result, expected)  # correctness
~~~

Reply:
you can parametrize this test by passing these functions in as well

something like

```
@pytest.mark.parametrize("f, f_numpy", [('sum', np.sum), ('mean', np.mean'), ('std', lambda x: np.std(x, ddof=0))])
def test_pivot_func_string(self, f, f_numpy):
```

note that ``std`` will resolve to pandas ``std`` and so ``np.std`` needs handling here (for the test)

Reply:
Thank you, both,  I'll incorporate these changes